www-src/app/lib/android_calendar_handler.coffee

```diff
@@ -1,18 +1,118 @@
-log = require('./persistent_log')
+androidCalendarHelper = require "./android_calendar_helper"
+CozyToAndroidCalendar = require \
+        "../replicator/transformer/cozy_to_android_calendar"
+request = require "./request"
+log = require("./persistent_log")
     prefix: "AndroidCalendarHandler"
     date: true
-androidCalendarHelper = require '../lib/android_calendar_helper'
 
 module.exports = class AndroidCalendarHandler
 
-    get: (name) ->
-        log.info "get"
+    ACCOUNT:
+        accountType: 'io.cozy'
+        accountName: 'myCozy'
 
-    create: (doc) ->
-        log.info "change"
+    constructor: (@db, @config, @calendarSync) ->
```

Comment:
A cache would be usefull to improve speed. You may use a coffee 'class' attribute to avoid aliasing even multiples instances of this handler exists.